packages/apollo-gateway/src/__tests__/integration/networkRequests.test.ts

```diff
@@ -178,7 +178,7 @@ it.each([
   spyGetServiceDefinitionsFromStorage.mockRestore();
 });
 
-it('Rollsback to a previous schema when triggered', async () => {
+it.skip('Rollsback to a previous schema when triggered', async () => {
```

Comment:
`it.goaway`? 😄 😭  Just leaving this note here so we don't inadvertently commit it.

Reply:
See notes in bb83f17. We've decided to leave these skipped for now 😞